packages/manager/apps/pci-savings-plan/src/utils/test/setupTests.tsx

```diff
@@ -0,0 +1,14 @@
+import { vi } from 'vitest';
+import React from 'react';
+import '@testing-library/jest-dom';
+
+vi.mock('react-i18next', () => ({
+  useTranslation: () => ({
+    t: (key: string) => key,
+  }),
+  Trans: (props: any) => (
+    <span data-testid="trans">
+      {props.i18nKey} {JSON.stringify(props.values)}
+    </span>
```

Comment:
Prefer checking if the translation function has been called with specific parameters instead